src/diffusers_interpret/token_attributions.py

```diff
@@ -0,0 +1,60 @@
+from typing import Tuple, List, Any, Iterable, Union
+import matplotlib.pyplot as plt
+
+
+class TokenAttributions(list):
+    def __init__(self, token_attributions: List[Tuple[str, float]]) -> None:
+        super().__init__(token_attributions)
+        self.token_attributions = token_attributions
+
+    def __getitem__(self, item: Union[str, int]) -> Any:
+        return getattr(self, item) if isinstance(item, str) else self.token_attributions[item]
+
+    def __setitem__(self, key: Union[str, int], value: Any) -> None:
+        setattr(self, key, value)
+
+    def plot(self, plot_type: str = 'barh', title: str = 'Token Attributions', **plot_kwargs) -> None:
+        '''
+        Plot the token attributions to have a comparative view.
+        Available plot types include bar chart, horizontal bar chart, and pie chart.
+        '''
+        tokens, attributions = list(zip(*self.token_attributions))
+
+        # get arguments from plot_kwargs
+        xlabel = plot_kwargs.get('xlabel')
+        ylabel = plot_kwargs.get('ylabel')
+        title = plot_kwargs.get('title') or title
```

Comment:
This is honestly a great way for me to learn how to use `**kwargs` through implementation! Thank you once again for the guidance ☺️
Since the parameter `title` is set in `def plot()` already, would this line be redundant? Line 60 by itself should be enought right?

Reply:
Yeah, good catch. But actually, there could be a corner case.
If I call `self.plot('barh', 'first title in args', title='second title in kwargs')`, this line 26 is actually helping. Maybe we can remove `title: str = 'Token Attributions'` from the arguments to avoid these bad usage